Makefile

```diff
@@ -95,33 +95,48 @@ test:
 MATCH=v.+\/mattermost-load-test-ng-v.+-linux-amd64.tar.gz
 REPLACE=$(NEXT_VER)\/mattermost-load-test-ng-$(NEXT_VER)-linux-amd64.tar.gz
 TAG_EXISTS=$(shell git rev-parse $(NEXT_VER) >/dev/null 2>&1; echo $$?)
+BRANCH_NAME=bump-$(NEXT_VER)
+PR_URL=https://github.com/mattermost/mattermost-load-test-ng/compare/master...$(BRANCH_NAME)?quick_pull=1&labels=2:+Dev+Review
+
+prepare-release:
+ifndef NEXT_VER
+	@echo "Error: NEXT_VER must be defined"
+else
+ifeq ($(TAG_EXISTS), 0)
+	@echo "Error: tag ${NEXT_VER} already exists"
+else
+	git checkout -b $(BRANCH_NAME) origin/master
```

Comment:
Maybe do a check if $(BRANCH_NAME) already exists?

Reply:
Good catch, thanks! Done in https://github.com/mattermost/mattermost-load-test-ng/pull/502/commits/7ec8af8a12209922a34b55999f5c82a07ce9fb4f